content/messaging/references/default-properties/index.mdx

```diff
@@ -0,0 +1,495 @@
+Specifies the ID of a particular greeting instance sent to the Customer. [More on greetings...](/messaging/customer-chat-api/#accept-greeting)
+
+|               |                                                                                                                 |
+| ------------- | --------------------------------------------------------------------------------------------------------------- |
+| **Access**    | <ul><li>for Agents, `r` in threads and events</li><li>for Customers, `w` in threads and `r` in events</li></ul> |
+| **Data type** | `string`                                                                                                        |
+
+### prechat_group_id
+
+Specifies the group ID in `prechats` with the `group_chooser` field. [More on form fields...](/messaging/customer-chat-api/#form-fields)
+
+|               |                             |
+| ------------- | --------------------------- |
+| **Access**    | `w` for Customers in events |
+| **Data type** | `int`                       |
+
+### welcome_author_id
+
+Specifies the ID of the welcome message author.
+
+|               |                                         |
+| ------------- | --------------------------------------- |
+| **Access**    | `rw` for Customers in chats and threads |
+| **Data type** | `string`                                |
+
+### welcome_message
+
+Specifies if a welcome message was sent to the Customer.
+
+|               |                                        |
+| ------------- | -------------------------------------- |
+| **Access**    | `r` for Agents and Customers in events |
+| **Data type** | `bool`                                 |
+
+### welcome_text
+
+Contains the welcome message text. Appears only if `"welcome_message": true`.
+
+|               |                                         |
+| ------------- | --------------------------------------- |
+| **Access**    | `rw` for Customers in chats and threads |
+| **Data type** | `string`                                |
+
+</Text>
+<Code>
+<CodeResponse title={'Properties in the lc2 namespace'}>
+
+```json
+{
+  "properties": {
+      "lc2": {
+          "file_safety_confirmation": true,
+          "form_type": "prechat",
+          "greeting_id": 2,
+          "greeting_unique_id": "Q10X0W041P",
+          "prechat_group_id": 0,
+          "welcome_author_id": "bbb67d600796e9f277e360e842418833",
+          "welcome_message": true,
+          "welcome_text": "Welcome! How can I help you today?"
+      }
+  }
+}
+```
+
+</CodeResponse>
+
+</Code>
+</Section>
+
+<Section>
+<Text>
+
+## rating
+
+The properties describing chat rating.
+
+### comment
+
+Contains the text message rating left by the Customer.
+
+|               |                                                                     |
+| ------------- | ------------------------------------------------------------------- |
+| **Access**    | `r` for Agents and `rw` for Customers in chats, threads, and events |
```

Comment:
it's inconsistent with other props, they use separate bullet points for agents and customers